### ISSUE TITLE
Unable to open screenshot

### ISSUE THREAD

**Issue by Amrkamel1** [Shelex/cypress-allure-plugin#12]
**Describe the bug**

allure-results folder has the screenshot but its not available on the html report as shown below

![image](https://user-images.githubusercontent.com/32042806/84387911-00ae1500-abf4-11ea-9573-c2c7619070a3.png)

![image](https://user-images.githubusercontent.com/32042806/84387981-1de2e380-abf4-11ea-86f7-580c37b2c230.png)

![image](https://user-images.githubusercontent.com/32042806/84388272-92b61d80-abf4-11ea-8861-6fb2fec5e40b.png)

**Expected behavior**
Should be able to see the screenshot

**Environment (please complete the following information):**
- Cypress version:                               4.8
- OS:                                                    windows

** Developer Tools Console Output **

**Comment by Amrkamel1** [comment]
issue doesn't exist on ubuntu

**Comment by Shelex** [comment]
@Amrkamel1 
 - what is your allure version?
 - do you open report with `allure open`?
 - does screenshot display 0B size in allure report?
 - do you use custom allure results folder?

**Comment by Amrkamel1** [comment]
> @Amrkamel1
> 
> * what is your allure version?
> * do you open report with `allure open`?
> * does screenshot display 0B size in allure report?
> * do you use custom allure results folder?

- allure CLI version is 2.13.0
- im using allure generate, it generates the html report but with no screenshot as shown in the screenshot above however same project is working fine and generates html with screenshot on ubuntu 
i have tried allure generate, allure open, allure serve all gives same result screenshot is not clickable
- yes screenshot appears with size 0B as shown in screenshot above 
i just use this properties below in cypress.json

```
  "env":
  {
    "allureResultsPath": "report/allure-results",
    "allure": true
  }
```

**Comment by Amrkamel1** [comment]
@Shelex  i have just update allure cmd to 12.13.2 but no luck same issue

**Comment by archital** [comment]
> allure CLI version is 2.13.0

Latest allure CLI  as well as Jenkins CI Allure plugin for me show exactly the same issue. 
My results folder is:` "allureResultsPath": "allure",`

**Comment by ridersaitou** [comment]
I also observed same issue 
Win 10 env + Cypress : 4.10 + Allure 12.3

step 1 : 
yarn cy:run --headless --> generate allure-results/ (this directory includes all attachment which is correct, png files also stored there)
step 2 :
yarn allure:report --> generate allure-report/ (in data/attachment/ there is only file csv there, no png files)
step 3 :
allure open --> report generated, there is a attachment name, but cannot be clicked, file size 0kb -- please see image below
 
![image](https://user-images.githubusercontent.com/7522326/87856842-ff3adf80-c94b-11ea-991d-74275c8b13a1.png)

**Comment by Shelex** [comment]
@Amrkamel1 @archital @ridersaitou 
just published v1.7.5 where trigger for screenshot processing has beed changed. Could you please check if issue is solved on your end?

**Comment by archital** [comment]
It still reproduced for me with v1.8.0
![Screenshot_5](https://user-images.githubusercontent.com/8686909/88227759-c44fe900-cc76-11ea-99bd-de03208fff2f.png)

**Comment by ridersaitou** [comment]
I'm still having same issue, not yet fixed in Windows 64
I also update Cypress to 4.11

I'm aware that several package are not compatible, dunno if it's related
![image](https://user-images.githubusercontent.com/7522326/88244254-75797200-ccbd-11ea-87c7-4475b4f0bdf4.png)

Report
![image](https://user-images.githubusercontent.com/7522326/88244292-904be680-ccbd-11ea-8fa6-0fc54e22f266.png)

**Comment by ahoad** [comment]
I've just been setting this up and have the same problem.

- On windows
- Cypres version: 4.12.1
- Allure version: 2.13.5
- @shelex/cypress-allure-plugin version: 1.8.0

Videos are attaching fine to the report
Screenshots are stored within the allure-results but then not in the attachments of the report

**Comment by SlElegant** [comment]
I have the same problem

On windows 10
Cypres version: 4.12.1
Allure version: 2.13.5
@shelex/cypress-allure-plugin version: 1.8.0

**Comment by Shelex** [comment]
@Amrkamel1 @archital @ridersaitou @ahoad @SlElegant Hello!
Finally got time and windows laptop to reproduce this issue and debug. 
Basically, problem was with detecting screenshot filename and behaviour of node `path.basename()`. As mentioned in [node.js docs](https://nodejs.org/api/path.html#path_windows_vs_posix) it might return different results, so I have added separate handling for windows screenshot paths. 

Could you please verify on your end that version 1.8.1 fixes this issue?

**Comment by archital** [comment]
@Shelex Thank you. The issue is fixed with the latest 1.8.1

**Comment by ridersaitou** [comment]
Hi @Shelex I confirm this issue fixed with latest 1.8.1 
Great Support, Thank you

![image](https://user-images.githubusercontent.com/7522326/90612248-faf73000-e231-11ea-9f7a-3092f715a419.png)
